app/cypress/integration/StreamPage.spec.js

```diff
@@ -173,8 +180,9 @@ describe('Stream read-only page (no edit permission)', () => {
                 status: 501,
                 response: {},
             })
+            const encodedId = encodeURIComponent(streamId)
             cy.route('GET', `/api/v1/streams/${streamId}`).as('getStream')
```

Comment:
FYI, `cy.route` expects decoded strings:
https://github.com/cypress-io/cypress-documentation/issues/1806

For `cy.visit` need to encode though